### ISSUE TITLE
Input fields - make Home/End buttons work

### ISSUE THREAD

**Issue by stonecrusher** [Woundorf/foxreplace#276]
If I want to edit a string, the string is selected completely in the first place.

So instead of grabbing the mouse again, my routine is to press `End` and start typing.
Oddly pressing `End` has no effect in foxreplace, so I end up overwriting the whole string, which is annoying.

**Comment by Woundorf** [comment]
You're right, I will fix it as soon as possible. In the meantime you can use the right arrow key to obtain the same effect the the End key should have.

**Comment by navid-zamani** [comment]
Hey, the home and end keys still don’t work at all.

I don’t even know how this is possible… That’s built-in functionality that requires no effort. It fact it takes effort, to make it not work. So in fact, it should be solvable by *removing* code. :)

The add-on is quite nice, but this drives me up the wall, which is quite unfortunate, because it wrecks my view of something actually good. If you are very busy, maybe I’ll dive into editing it myself. But I’m quite swamped myself right now., so it’s unlikely. 
(I wish us all less hassle and more time. :)

**Comment by Woundorf** [comment]
I wish it was as easy as removing code. The issue is related to the ag-grid library used, that has an event handler. Maybe it is not used correctly or maybe it was a bug or limitation of the library at the time (I updated it last time 8 years ago). It works as expected in their examples here: https://www.ag-grid.com/javascript-data-grid/cell-editing/.

>  (I wish us all less hassle and more time. :)

I agree :)
